static/themes/default.css

```diff
@@ -0,0 +1,31 @@
+.resume {
+  --font-size: calc(var(--resume-ratio) * 16px);
```

Comment:
I hate that I have to multiply each number value by the `--resume-ratio` to make sure everything is aligned properly when it shrinks down, but it is what it is.

This means that I need to fill in some default values for things like padding and margins in `ul`s and `li`s and stuff so it's consistent between all of the sizes. I should make a todo comment for that.

Reply:
done